decentdao/fractal-interface
# Update deterministic avatar generation library

Replaces the "Jazzicon" avatars with classic "Blockie"-style avatar, to be consistent with the ecosystem.

Testing

1. Connect an address to Fractal
2. See how the new icon that shows up for your address
3. Is the same generated icon that shows up for your address on Etherscan
4. And Metamask

Fractal:
<img width="293" alt="Screenshot 2024-04-25 at 8 17 44 PM" src="https://github.com/decentdao/fractal-interface/assets/706929/53d05ea1-296b-4c3f-bb9c-9901a54f7f88">

Etherscan:
<img width="360" alt="Screenshot 2024-04-25 at 8 17 56 PM" src="https://github.com/decentdao/fractal-interface/assets/706929/25ac683f-f326-4216-a770-ba5ba85290ce">
